pkg/cmd/cli/cmd/rsync/copyrsync.go

```diff
@@ -28,7 +28,7 @@ type rsyncStrategy struct {
 	podChecker     podChecker
 }
 
-var rshExcludeFlags = sets.NewString("delete", "strategy", "quiet", "include", "exclude", "progress", "no-perms")
+var rshExcludeFlags = sets.NewString("delete", "strategy", "quiet", "include", "exclude", "progress", "no-perms", "watch")
```

Comment:
There are comments referring to this variable in different places, but no inline documentation... perhaps it's a good opportunity to add it?